acropolis-persistence/src/main/kotlin/org/ephyra/acropolis/persistence/api/entity/ApplicationSoftwareEntity.kt

```diff
@@ -16,15 +16,15 @@ import javax.persistence.*
  *  - An interactive shell or terminal
  */
 @Entity
-data class ApplicationSoftwareEntity @JvmOverloads constructor (
-    @Column(nullable=false)
-    var name: String,
+data class ApplicationSoftwareEntity @JvmOverloads constructor(
+        @Column(nullable = false)
```

Comment:
I swear there is a bug in the formatter for Kotlin. It sometimes leaves these and sometimes indents them more. Oh well, doesn't make much difference :)

Reply:
Aye , Intellij always does it for me. No use fighting the inevitable